lsp.el

```diff
@@ -488,7 +514,9 @@ INHERIT-INPUT-METHOD will be proxied to `completing-read' without changes."
   ;; identifiers it supports or expects.
   (language-id nil :read-only t)
 
-  (DEPRECATED_send-sync nil :read-only t)
+  ;; ‘add-on?’ when set to t this will indicate that the server can be started as
+  ;; a additional to another server. In case there are several
+  (add-on? nil :read-only t)
```

Comment:
The parameter name and its association comments are still obscure to me.

Reply:
Oh, the comment is not finished. When the server has add-on? = true this means that it will be started along the other servers for that mode. For example, spring-boot-language-server could run in parallel with jdt-ls. I am open for suggestions for better naming.